### ISSUE TITLE
Android 2.3 not recognize tests 

### ISSUE THREAD

**Issue by brescia123** [kotlintest/kotlintest#157]
Android Studio 2.3 introduced a new "Android JUnit" configuration in place of the common "JUnit" one. (https://code.google.com/p/android/issues/detail?id=230419#c5) 
This new configuration doesn't discover tests produced through the library.

**Comment by elect86** [comment]
Try to add this line before your test class, Giacomo

```
@RunWith(KTestJUnitRunner::class)
class quaternion : StringSpec() {
```

**Comment by brescia123** [comment]
It works, thanks! Does this mean that I have to add the annotation to every test class?

**Comment by elect86** [comment]
Till we will be gifted with Kotlintest 2.0, yes :p

Although it has been a fluctuating behaviour for me, some classes works without, some others need that

**Comment by brescia123** [comment]
So the fix should be kotlintest generating JUnit test with the annotation?

**Comment by elect86** [comment]
No idea, @sksamuel may answer that

**Comment by helmbold** [comment]
The fix is to annotate the spec classes (the base classes for your test) with `@RunWith`. I've already done this in KotlinTest 2.0.

**Comment by sksamuel** [comment]
So we're happy this is fixed in KotlinTest 2.0 + @helmbold  ?

**Comment by helmbold** [comment]
Yes, it should be fixed.
